corehq/apps/accounting/migrations/0017_nonnullable_char_fields.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.11 on 2018-03-13 17:56
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounting', '0016_grandfather_reportbuilder_5_pro'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='subscription',
+            name='no_invoice_reason',
+            field=models.CharField(blank=True, default='', max_length=256),
```

Comment:
Is this enough to convert existing NULLs to empty strings? Or are there currently no NULLs?

Reply:
> Is this enough to convert existing NULLs to empty strings?

Correct

Reply:
I don't know if that's true. When deploying ICDS I ran into a similar error to this: https://stackoverflow.com/questions/12838111/south-cannot-alter-table-because-it-has-pending-trigger-events (that's for south, but it's the same thing)

I solved it by manually running:

```commcarehq=> UPDATE accounting_subscription SET salesforce_contract_id = '' WHERE salesforce_contract_id IS NULL;
UPDATE 55
commcarehq=> UPDATE accounting_subscription SET no_invoice_reason = '' WHERE no_invoice_reason IS NULL;
UPDATE 55
```

Reply:
@emord I was able to reproduce this locally.  Added a fix: https://github.com/dimagi/commcare-hq/pull/19801